Package.swift

```diff
@@ -18,7 +18,7 @@ let package = Package(
         ),
         .package(
             url: "https://github.com/rewardStyle/PryntTrimmerView",
-            .exact("4.0.2")
+            .branch("CC-505-creators-can-view-video-lengths-when-editing-in-storytelling-posts")
```

Comment:
Just putting a note here so that we'll remember to switch this back once we release the new version of PryntTrimmerView. That PR approved so we could probably release it already?

Reply:
@iirovi-rs Updated here: https://github.com/rewardStyle/YPImagePicker/pull/34/commits/cb6feb2a13233edc8fe40452f6559ca34224004e